pkg/controller/database/claim.go

```diff
@@ -96,27 +96,29 @@ type PostgreSQLInstanceClaimController struct{}
 func (c *PostgreSQLInstanceClaimController) SetupWithManager(mgr ctrl.Manager) error {
 	name := strings.ToLower(fmt.Sprintf("%s.%s.%s",
 		databasev1alpha1.PostgreSQLInstanceKind,
-		v1alpha2.RDSInstanceKind,
-		v1alpha2.Group))
+		v1beta1.RDSInstanceKind,
+		v1beta1.Group))
 
 	r := resource.NewClaimReconciler(mgr,
 		resource.ClaimKind(databasev1alpha1.PostgreSQLInstanceGroupVersionKind),
-		resource.ClassKind(v1alpha2.RDSInstanceClassGroupVersionKind),
-		resource.ManagedKind(v1alpha2.RDSInstanceGroupVersionKind),
+		resource.ClassKind(v1beta1.RDSInstanceClassGroupVersionKind),
+		resource.ManagedKind(v1beta1.RDSInstanceGroupVersionKind),
+		resource.WithManagedBinder(resource.NewAPIManagedStatusBinder(mgr.GetClient())),
```

Comment:
This signature changed after my static provisioning work; it's now `(mgr.GetClient(), mgr.GetScheme())`

Reply:
Thanks for the heads-up! I'll update the crossplane-runtime in a final commit.